src/dotnet/commands/dotnet-pack3/Pack3Command.cs

```diff
@@ -30,6 +30,12 @@ public static int Run(string[] args)
             var noBuild = cmd.Option("--no-build",
                 "Do not build project before packing", 
                 CommandOptionType.NoValue);
+            var includeSymbols = cmd.Option("--include-symbols",
+                "Include PDBs along with the DLLs in the output folder",
+                CommandOptionType.NoValue);
+            var includeSource = cmd.Option("--include-source",
+                "Include PDBs and source files. Source files go into the src folder in the resulting nuget package",
```

Comment:
Is this description correct?  Why would `--include-source` mean that I should include PDBs?  Why not have the user specify both `--include-symbols` and `--include-source` if they want both?